worlds/alttp/Options.py

```diff
@@ -27,6 +28,29 @@ class Goal(Choice):
     option_hand_in = 2
 
 
+class OpenPyramid(Choice):
+    """Determines whether the hole at the top of pyramid is open.
+    Goal will open the pyramid if the goal requires you to kill Ganon, without needing to kill Agahnim 2
+    Auto is the same as goal except if Ganon's hole is in another location will be closed"""
+    display_name = "Open Pyramid Hole"
+    option_goal = 0
```

Comment:
this makes goal the default, is that consistent with how it used to be? gut feeling would say auto should be default.

Reply:
i copied them in the order they were in in the yaml but i'm more than willing to swap them around